Magento2/Sniffs/Legacy/AbstractBlockSniff.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Sniffs\Legacy;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class AbstractBlockSniff implements Sniff
+{
+    const CHILD_HTML_METHOD = 'getChildHtml';
```

Comment:
Please use `private` access level by default
```suggestion
    private const CHILD_HTML_METHOD = 'getChildHtml';
```

Reply:
Done